prov/usnic/src/usdf.h

```diff
@@ -295,6 +295,12 @@ struct usdf_ep {
 			int ep_sock;
 			struct usdf_av *ep_av;
 
+			/*
+			 * Move to ep_tx and ep_rx
```

Comment:
Please throw a `TODO` in that comment so that it's clearer.  Also, there's a lot of vertical space unnecessarily being used on the line above and below the text.  A single line `/* TODO: [...] */` comment would be better, IMO.